mssqlcli/mssqlclioptionsparser.py

```diff
@@ -15,8 +15,7 @@
 def create_parser():
     args_parser = argparse.ArgumentParser(
         prog=u'mssql-cli',
-        description=u'Microsoft SQL Server CLI. ' +
-        u'Version {}'.format(mssqlcli.__version__))
+        description=u'Microsoft SQL Server CLI. Version {}'.format(mssqlcli.__version__))
 
```

Comment:
Do we need to change this file? If not, let's revert. We have lots of flake8 and formatting issues but let's address those in other PRs.

Reply:
reverted